### PR TITLE
Feature/paragraph updates

### PR DESCRIPTION
**Pull request by Sqrcz**
Update for point 1. on **[List of random issues](https://github.com/themesberg/flowbite-vue/discussions/236)** 

- `class` prop added
- `useMergeClasses()` used to manage user classes
- added `defaultClasses` property with default classes: `mb-3 last:mb-0`
- `color` prop removed (it was just an unnecessary wrapper/middleware for tailwind classes)
- `color` default value moved to `defaultClasses`

_As now with `useMergeClasses()` user can easily set them... I would like to remove as well other props that exist only for setting tailwind classes.
If you agree I'll remove them in the next commit._

### CODE REVIEW

File: docs/pages/getting-started.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ npm i flowbite flowbite-vue
 ```javascript
 module.exports = {
   content: [
-    'node_modules/flowbite-vue/**/*.{js,jsx,ts,tsx}',
+    'node_modules/flowbite-vue/**/*.{js,jsx,ts,tsx,vue}',
```
Comment (Sqrcz) [inline docs/pages/getting-started.md:22]: this change was required for custom classes like `last:mb-0` to be visible in projects using `flowbite-vue` (without it custom classes like that was not discovered by Tailwindcss.

File: src/composables/useMergeClasses.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 import { twMerge } from 'tailwind-merge'
 
-export const useMergeClasses = (componentClasses: string): string =>
+export const useMergeClasses = (componentClasses: string | string[]): string =>
```
Comment (Sqrcz) [inline src/composables/useMergeClasses.ts:3]: `twMerge` accepts both types... so we shouldn't limit those here.

**Review by cogor — APPROVED** [review]

**Comment by cogor** [comment]
LGTM

**Comment by Sqrcz** [comment]
OK, I've removed all props as those were just class wrappers.

File: docs/components/typography/p/FwbPExampleCustom.vue [hunk 3]
```diff
@@ -1,7 +1,40 @@
...
+      tools, so your IT support and operations teams have richer contextual
+      information to rapidly respond to requests, incidents, and changes.
+    </fwb-p>
+    <fwb-p class="font-bold">
+      Deliver great service experiences fast - without the complexityof
+      traditional ITSM solutions. Accelerate critical development work,
+      eliminate toil, and deploy changes with ease, with a complete
+      audit trail for every change.
+    </fwb-p>
+    <fwb-p class="test-left">
```
Comment (cogor) [inline docs/components/typography/p/FwbPExampleCustom.vue:15]:
```suggestion
    <fwb-p class="text-left">
```

File: docs/components/typography/p/FwbPExampleCustom.vue [hunk 4]
```diff
@@ -1,7 +1,40 @@
...
+      Deliver great service experiences fast - without the complexityof
+      traditional ITSM solutions. Accelerate critical development work,
+      eliminate toil, and deploy changes with ease, with a complete
+      audit trail for every change.
+    </fwb-p>
+    <fwb-p class="test-center">
```
Comment (cogor) [inline docs/components/typography/p/FwbPExampleCustom.vue:21]:
```suggestion
    <fwb-p class="text-center">
```

**Pull request merged**